CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This file is used to list changes made in each version of the haproxy cookbook.
 ## v2.0.3 (tbc)
 
 - Configurable debug options
+- Merging [#123][] - CentOS7 compatibility
```

Comment:
This should probably get expanded to include at least a mention of using poise now. People have strong feelings both ways about poise.